webpack.config.js

```diff
@@ -65,18 +103,39 @@ const prod = {
           })
         })
       }
-    }
+    },
+    new CompressionPlugin({
+      algorithm: 'gzip',
+      test: /\.(js|css|svg)$/,
+      exclude: /.map$/,
+      compressionOptions: {
+        level: 9,
+        numiterations: 15,
+        minRatio: 0.8
+      }
```

Comment:
Let's remove `CompressionPlugin` from this project.

We don't benefit from this at all, requires extra configuration ([`gzip_static on`](https://nginx.org/en/docs/http/ngx_http_gzip_static_module.html)), and these unused `.gz` files only makes pinned DAG bigger. 

FYSA gzip compression already works, but it happens outside gateway, on the load balances / reverse proxies in front of it:

```console
$ curl -D - -o /dev/null https://inbrowser.link/ -v --compressed
[..]
* [HTTP/2] [1] [accept-encoding: deflate, gzip, zstd]
[..]
< content-encoding: gzip
```